docs/content/en/docs/concepts/api/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Skaffold API"
+linkTitle: "Skaffold API"
+weight: 40
+---
+
+When using `skaffold dev` or `skaffold debug`, Skaffold exposes a API server over its lifetime.
+The API server is the primary way tools like IDEs to integrate with skaffold and subscribe to events 
+regarding the different phases in a pipeline run and to get the overall state of the pipeline.
+The API server also provides fine grain controls to Skaffold's individual components:
+build, deploy and sync, as opposed to relying on Skaffold’s built-in trigger mechanisms.
+
+
+## Skaffold API 
+Skaffold API is restful and `gRPC` based, so it works with any language that has an HTTP library, such as cURL and urllib.
```

Comment:
```suggestion
The Skaffold API is both restful and `gRPC` based, so it can be accessed directly through a terminal via `curl`, or through your favorite client library for issuing HTTP requests or writing `gRPC` clients.
```